starter-code/main.js

```diff
@@ -53,7 +57,188 @@ mongoClient.connect(`mongodb://localhost:27017/crunchbase`, (error, db) => {
+                console.log(error);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              } else {
+                console.log(result);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              }
+            })
+            break;
+
+          case "8":
+            // 8.- Find the company with the name "Facebook"
+            db.collection('companies').find({ name: "Facebook" }, { name: 1, category_code: 1, description: 1, _id: 0 }).toArray((error, result) => {
+              if (error) {
+                console.log(error);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              } else {
+                console.log(result);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              }
+            })
+            break;
+
+          case "9":
+            // 9.- How many employees has Facebook?
+            db.collection('companies').find({ name: "Facebook" }, { number_of_employees: 1, _id: 0 }).toArray((error, result) => {
+              if (error) {
+                console.log(error);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              } else {
+                console.log(result);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              }
+            })
+            break;
+
+          case "10":
+            // 10.- List the name of all the products of Facebook
+            db.collection('companies').find({ name: "Facebook" }, { "products.name": 1, _id: 0 }).toArray((error, result) => {
+              if (error) {
+                console.log(error);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              } else {
+                console.log(result);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              }
+            })
+            break;
+
+          case "11":
+            // 11.- List the people that are working at Facebook right now (check relationships field)
+            db.collection('companies').find({ $and: [{ name: 'Facebook' }, { "relationships.is_past": false }] }, { "relationships.is_past": 1, _id: 0 }).toArray((error, result) => {
+              if (error) {
+                console.log(error);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              } else {
+                console.log(result);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              }
+            })
+            break;
+
+          case "12":
+            // 12.- List all the companies where "david-ebersman" has worked.
+            db.collection('companies').find({ "relationships.person.permalink": "david-ebersman" }, { name: 1, _id: 0 }).toArray((error, result) => {
```

Comment:
Si no conocieramos el formato del permalink podríamos hacer:
`{ $and: [{ 'relationships.person.first_name': 'David', 'relationships.person.last_name': 'Ebersman' }] },`